docs/reference/commandline/plugin_create.md

```diff
@@ -36,16 +36,14 @@ The following example shows how to create a sample `plugin`.
 
 ```bash
 $ ls -ls /home/pluginDir
-
+total 4
 4 -rw-r--r--  1 root root 431 Nov  7 01:40 config.json
 0 drwxr-xr-x 19 root root 420 Nov  7 01:40 rootfs
 
 $ docker plugin create plugin /home/pluginDir
-
```

Comment:
Empty lines are added consistently across all the files in the `commandline` directory